pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/internal/pex/PipFreezeAction.java

```diff
@@ -50,7 +50,14 @@ public PipFreezeAction(Project project) {
         developmentDependencies.addAll(configurationToSet(project, PythonPlugin.CONFIGURATION_TEST));
 
         // Special cases, such as sphinx-rtd-theme with weird metadata
-        developmentDependencies.addAll(Arrays.asList("sphinx-rtd-theme", "sphinx_rtd_theme"));
+        developmentDependencies.addAll(Arrays.asList(
```

Comment:
What do you think we should do to handle this gracefully into the future? It'd be painful to have to maintain a list like this indefinitely, although at the moment we don't have any other options. Just curious if you have ideas.